docs/resources/virtual_environment_file.md

```diff
@@ -11,6 +11,10 @@ subcategory: Virtual Environment
 
 Manages a file.
 
+> This resource uses SSH access to the node. You might need to configure the [`ssh` option in the `provider` section](../index.md#node-ip-address-used-for-ssh-connection).
+
+> `content_type=snippets` requires configuration in the 'datacenter>storage' section of proxmox before first use. The content type needs to be selected. 
+
```

Comment:
SSH is used primarily for `snippet` content type. `iso` and `vztmpl` are [using](https://github.com/bpg/terraform-provider-proxmox/blob/2a56c23f52abda293f328196a0d80b9becd749a7/proxmoxtf/resource/file.go#L523-L525) PVE API.

Does it make sense to wrap this note into a separate "Snippets" section? Either here, before the "Example Usage", or below, beside the "Important Notes".

The last line here is kind of duplicates the existing note on line 114 below. Disabled snippets is very common, but the same issue could happen with any content type on any datastore. Though I'm totally fine to re-iterate that explicitly for snippets :)

Reply:
Ah, yes I saw that in the code too! (Is it right thaz the check is only executed for content types not beeing 'iso or ' vztmpl'?) 

I will revise this PR to not duplicate this information. 

Would you be fine with a section for each content type? Than I would give an example per content type and state when it is not enabled by default. 
Linking to the important notes for data types using ssh.

Reply:
> Ah, yes I saw that in the code too! (Is it right that the check is only executed for content types not beeing 'iso or ' vztmpl'?)

Yes, but realistically, only the `snippet` content type is what's left (and what's using SSH for upload). 
Not sure if anyone is using `backup`, so that was my reasoning behind crating a section only for snippets. But if you're planning to add examples for each content type, that would be great!

Reply:
I added a section for each type and added info boxes depending if they use ssh or are enabled by default.